cmd/connectconformance/main.go

```diff
@@ -33,6 +33,9 @@ const (
 	configFlagName        = "conf"
 	testFileFlagName      = "test-file"
 	knownFailingFlagName  = "known-failing"
+	knownFlakyFlagName    = "known-flaky"
+	runFlagName           = "run"
+	noRunFlagName         = "no-run"
```

Comment:
I feel like `skip` would be slightly clearer, but I'm not hellbent on it 🤷‍♂️

Reply:
I debated that. I was looking for something that was obviously an antonym for `--run`. I considered `--include` and `--exclude`, but was worried those words weren't as clear/obvious when it comes to what these flags were for. But I can see "skip" also being obvious, even if the word by itself isn't related to the word "run".

Another thing that concerned me a little bit was how this now differs from the `--known-failing` flag, which names a file instead of being an actual pattern that can be specified multiple times. Maybe I should change them all to be consistent so that all of `--known-failing`, `--known-flaky`, `--run`, and `--skip` take a _pattern_ as their argument and can be specified multiple times. But you can prefix the argument with `@` to indicate the name of a file to read. So existing usages would have to add `@` to their `--known-failing` arguments, but then everything is more consistent.

@smaye81, WDYT?

Reply:
Sounds good to me. We'd only have one place (grpc-web) to fix the usage of `--known-failing` so that should be easy.

Reply:
> We'd only have one place (grpc-web) to fix the usage of --known-failing so that should be easy.

It's actually used in connect-kotlin and connect-swift, too. But still easy to update.